### ISSUE TITLE
popular languages

### ISSUE THREAD

**Issue by tablatronix** [GetSimpleCMS/GetSimpleCMS#1199]
http://get-simple.info/forums/showthread.php?tid=5505&pid=59046#pid59046

Top languages over past several months.

- German Language File GS 3.3 (aileesh)
- Russian Language for GetSimple CMS (mikhey)
- Czech language (zelvik)
- Spanish language for GetSimple (Carlos)
- Hungarian lang file / Magyar nyelvi fájl (rg6julian)
- Polish language (spolszczenie) GS 3.3 (yojoe)
- FR Traduction en francais pour v.3.3 : French translation for 3.3-ready (emanwebdev)

**Comment by tablatronix** [comment]
German Language File GS 3.3 by (aileesh) http://get-simple.info/extend/language/german-language-file-gs-33/762/ 
Russian Language for GetSimple CMS by (mikhey) http://get-simple.info/extend/language/russian-language-for-getsimple-cms/704/ 
Czech language by (zelvik) http://get-simple.info/extend/language/czech-language/987/ 
Spanish language for GetSimple by (Carlos) http://get-simple.info/extend/language/spanish-language-for-getsimple/338/ 
Magyar nyelvi fájl by (rg6julian) http://get-simple.info/extend/language/hungarian-lang-file-magyar-nyelvi-faacutejl/474/ Hungarian lang file / 
Polish language (spolszczenie) GS 3.3 by (yojoe) http://get-simple.info/extend/language/polish-language-spolszczenie-gs-33/272/ 
Czech language for 3.0 - 3.2 by (birkoff) http://get-simple.info/extend/language/czech-language-for-30-32/204/ 
FR Traduction en francais pour v.3.3 : French translation for 3.3-ready by (emanwebdev) http://get-simple.info/extend/language/fr-traduction-en-francais-pour-v33-french-translation-for-33-ready/753/

**Comment by tablatronix** [comment]
* add these to ckeditor install as well

**Comment by zelvik** [comment]
When I was creating czech language, I used previous versions which hadn't been working. Now, I updated it to actual version of GS again ;).
And czech in ckeditor by default I welcome, it will be simplier with this :). I'm not sure why, but CKEditor is very sensitive for version of language pack and now I must resolve it by using defined language in gsconfig as default for ckeditor (so I must edit edit.php a little, eventually see czech language pack ;))

**Comment by tablatronix** [comment]
Ckeditor must be built with languages since they are installed in all plugin folders. You cant just add a language later

**Comment by zelvik** [comment]
Yes, but when I added czech language, it didn't load it. I tried find official czech language for ckeditor of GS version, but I wasn't success. Maybe, I think, that ckeditor is checking integrity of language and when find some small detail (which I didn't found as differences between en and cz), it used instead prefered lang default languge. And for case, when you hardcode default language it use it, because it can't used another one. Do you think, that is it possible, or is problem in some another place?

**Comment by tablatronix** [comment]
Didnt load it in gs? Did not show up in settings? You need to be more specific and post steps of ehat you are talking about. I have no idea what you mean.

**Comment by zelvik** [comment]
When I put it to lang folder of ckeditor and set in gsconfig, it wasn't loaded in editor (when I opened or created a page). Language of GS own is ok and in settings can be set, but language of editor is second thing... And I think, that this two languages are independent, aren't?
